Realm/RLMSyncSession.h

```diff
@@ -25,7 +25,7 @@
  */
 typedef NS_ENUM(NSUInteger, RLMSyncSessionState) {
     /// The sync session is actively communicating or attempting to communicate
-    /// with MongoDB Realm. A session is considered Active even if
+    /// with Atlas App Services. A session is considered Active even if
```

Comment:
Here is where I would expect to see 'Atlas Device Sync', as this enum is directly referring to sync

Reply:
I see that as the details of protocol, rather than feature related